clients/drcachesim/tracer/tracer.cpp

```diff
@@ -971,11 +1213,18 @@ drmemtrace_client_main(client_id_t id, int argc, const char *argv[])
                droption_parser_t::usage_short(DROPTION_SCOPE_ALL).c_str());
         dr_abort();
     }
-    if (!op_offline.get_value() && op_ipc_name.get_value().empty()) {
-        NOTIFY(0, "Usage error: ipc name is required\nUsage:\n%s",
-               droption_parser_t::usage_short(DROPTION_SCOPE_ALL).c_str());
-        dr_abort();
-    } else if (op_offline.get_value() && op_outdir.get_value().empty()) {
+    if (!op_offline.get_value()) {
+        if (op_ipc_name.get_value().empty()) {
+            NOTIFY(0, "Usage error: ipc name is required\nUsage:\n%s",
+                   droption_parser_t::usage_short(DROPTION_SCOPE_ALL).c_str());
+            dr_abort();
+        }
+        if (op_num_threads.get_value() > 1) {
+            NOTIFY(0, "Usage error: only one sideline thread is supported "
+                   "for online simulation\n");
```

Comment:
Why?  Please explain in a comment: is this temporary or permanent?
Is this documented in options.cpp?